### PR TITLE
Add logging to mqtt telemetry

### PR DESCRIPTION
**Pull request by cloud-rocket**

### CODE REVIEW

**Comment by cloud-rocket** [comment]
@DocGarbanzo , can please merge it?
Tnx

**Comment by cloud-rocket** [comment]
ping @DocGarbanzo

File: donkeycar/parts/telemetry.py [hunk 1]
```diff
@@ -85,18 +103,41 @@ def publish(self):
...
+                        logger.error('Cannot publish topic "{}" with value of type {}'.format(topic, type(v)))
+                    except:
+                        logger.error('Error publishing topic "{}"'.format(topic))
+                    
+            # Publish all logs
+            for tm, sample in packet.items():
+                if LOG_MQTT_KEY in sample:
+                    try:
+                        self._mqtt_client.publish('{}/{}'.format(self._topic, LOG_MQTT_KEY), sample[LOG_MQTT_KEY])
+                    except:
```
Comment (DocGarbanzo) [inline donkeycar/parts/telemetry.py:139]: Better to print the exception here, so you know what went wrong:

```python
except Exception as e:
    logger.error(f'Error publishing log {topic}: {e}') 
```

**Review by DocGarbanzo — CHANGES_REQUESTED** [review]
Looks good in general, just small comments about printing and exception handling.

File: donkeycar/parts/telemetry.py (hunk 1 above)
Comment (cloud-rocket) [reply donkeycar/parts/telemetry.py]: fixed

File: donkeycar/parts/telemetry.py [hunk 2]
```diff
@@ -85,18 +103,41 @@ def publish(self):
...
+            
         if self._use_json_format:
             packet = [{'ts': k, 'values': v} for k, v in packet.items()]
             payload = json.dumps(packet)
 
-            self._mqtt_client.publish(self._topic, payload)
+            try:
+                self._mqtt_client.publish(self._topic, payload)
+            except:
+                logger.error('Error publishing topic "{}"'.format(topic))
```
[the remark this answers is not in the document]
Comment (cloud-rocket) [reply donkeycar/parts/telemetry.py]: fixed

**Review by DocGarbanzo — APPROVED** [review]
LGTM

**Pull request merged**